src/node/routes/vscode.ts

```diff
@@ -31,6 +32,30 @@ export class CodeServerRouteWrapper {
       })
     }
 
+    // Ew means the workspace was closed so clear the last folder/workspace.
+    const { query } = await settings.read()
+    if (req.query.ew) {
+      delete query.folder
+      delete query.workspace
+    }
+
+    // Redirect to the last folder/workspace if nothing else is opened.
+    if (
+      !req.query.folder &&
```

Comment:
note to self: would be good to write a unit test for this logic!

Reply:
ahaha was just thinking the same thing yesterday, I am going to write one today.

Reply:
Twins!